src/field.rs

```diff
@@ -624,6 +650,20 @@ pub fn random_vector<F: FieldElement>(len: usize) -> Result<Vec<F>, PrngError> {
         .collect())
 }
 
+/// Generate a vector of uniform random field elements.
+pub fn random_vector2<F: FieldElement>(len: usize) -> Result<Vec<F>, PrngError> {
+    Ok(Prng2::generate(Suite::Aes128CtrHmacSha256)?
+        .take(len)
+        .collect())
+}
+
+/// Generate a vector of uniform random field elements.
```

Comment:
This is https://github.com/cjpatton/vdaf/pull/29, right? Can you rename the function to something more meaningful, such as "random_vector_pad_then_reduce" or something?